plugins/sentry/config/initializers/sentry.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+if sentry_dns = ENV['SENTRY_DSN']
+  Sentry.init do |config|
+    config.dsn = sentry_dns
+    config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+    config.environment = Rails.env
+
+    # We recommend adjusting this value in production:
+    config.traces_sample_rate = 0
+  end
+
+  # make events clickable see https://github.com/getsentry/sentry-ruby/issues/1786
+  if project = ENV["SENTRY_PROJECT"]
+    Sentry::LoggingHelper.prepend(Module.new do
+      def log_info(message)
+        super(message.sub( /(\S+) to Sentry/, "https://sentry.io/organizations/#{project}/issues/?query=\\1 to Sentry"
```

Comment:
this won't work because `project` won't be in scope, so need to repeat the `ENV["SENTRY_PROJECT"]` or use a constant